src/dispatcher.js

```diff
@@ -60,36 +61,36 @@ class Dispatcher extends machina.BehavioralFsm {
 			actionMap: {},
 			states: {
 				ready: {
-					_onEnter: function() {
+					_onEnter: () => {
```

Comment:
I was honestly surprised arrow functions even worked here. I pulled up the file to refresh my memory... The only reason they work is because the `this` at the time of the arrow functions being defined is the instance being stood up by the constructor. On the whole, you want to avoid arrow fns as methods when `this` is used internally in the function body. If this object were, for whatever reason, defined outside the constructor and passed here, it would break. What you want to use here instead are shorthand methods...so:
```
ready: {
    _onEnter() {
        this.actionContext = undefined;
    }
}
```
Another thing to bear in mind is that by declaring these as arrow fns here, you prevent consumers from being able to call `.bind()` on the method (not that anything is doing that here, but it's important to keep the impact in mind).